app/views/admin/movies/_base_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= form_with url: build_admin_movies_path, class: 'form' do |f| %>
+  <div data-controller='movie' data-movie-url-value='<%= search_admin_movies_path %>'>
+    <div class='form__field form__field--full'>
+      <%= f.label :imdb_data, 'Search a Movie (Name* - Year)', class: 'form__label' %>
+      <%= f.select :imdb_data, {}, {include_blank: 'Select a movie'}, class: 'form__select', data: {movie_target: 'contentSearch', action: 'change->movie#submitForm'}%>
```

Comment:
for ```select``` elements, stimulus default action is ```change```. You could erase ```change->``` from your ```data-action```.
https://stimulus.hotwire.dev/reference/actions

Reply:
good catch! <3